doc/recognition.xml

```diff
@@ -416,6 +419,19 @@ elements that are generated from them remember, how they were acquired.
 </Description>
 </ManSection>
 
+<ManSection>
+<Attr Name="HomomRangeCheck" Arg="ri"/>
```

Comment:
I think a better name is `HomomDomainCheck`. 

In the case you describe below the domain of the homomorphism consists of those matrices that leave the corresponding subspace invariant. 
The new function checks whether its argument is in the domain of the homomorphism.

Reply:
my reason for using Range is (I think, I don't really use homomorphisms in GAP very much) that GAP uses Range(hom), rather than Domain(hom), so the name lines up (somewhat) with GAP. However, I'm happy to be (a) wrong, or (b) just more standard maths names.

Reply:
I checked the documentation. `Source` is the name of the source and `Range` the name of the target.

Reply:
Yes, that's just my inability to do.. words.

Reply:
thanks